libs/wallet-ui/src/components/network-compatibility-dialog/network-compatibility-dialog.tsx

```diff
@@ -64,15 +66,26 @@ const getTitle = (subview: Subview) => {
   }
 }
 
-const shouldBeOpen = (
-  networkData?: NetworkCompatibility,
-  telemetryConsentAsked?: boolean
-) => {
-  return telemetryConsentAsked === true && networkData?.isCompatible === false
+type GetShouldOpenProps = {
```

Comment:
**nitpick (non-blocking)**:  I got tripped up here by naming. I think the naming all makes sense taken as a whole, but I was wondering why `Features` went straight in to a hook for deciding if the network panel should open. Obviously reading the whole thing, I get it - one of the feature flags (and potentially more) control if this thing should show or not. 

It would be more obvious if the specific feature flag was passed in. But I don't think it's worth changing. It's pretty clear.

Reply:
yeah, i think it makes sense, what you said, might revisit it next time, clicked the merge way too quickly here :)